Woof/WoofTests/ViewModel/OwnerCardViewModelTests.swift

```diff
@@ -0,0 +1,34 @@
+import XCTest
+
+final class OwnerCardViewModelTests: XCTestCase {
+    func testOwnerCardViewModelExistsInAPI() {
+        let owner = Owner(
```

Comment:
Can we use dummy model in tests so we don't have to fill all these fields by hand?

Reply:
Yes, sure. Thants for the comment.
I've  deleted this file, because, as we discussed with the team, the viewModel here is not needed (and tests for it  also was deleted).